src/panels/lovelace/editor/config-elements/hui-picture-glance-card-editor.ts

```diff
@@ -0,0 +1,243 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+} from "lit-element";
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+
+import "../../components/hui-action-editor";
+import "../../components/hui-entity-editor";
+import "../../../../components/entity/ha-entity-picker";
+
+import { struct } from "../../common/structs/struct";
+import {
+  EntitiesEditorEvent,
+  EditorTarget,
+  actionConfigStruct,
+  entitiesConfigStruct,
+} from "../types";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { configElementStyle } from "./config-elements-style";
+import { ActionConfig } from "../../../../data/lovelace";
+import { PictureGlanceCardConfig } from "../../cards/types";
+import { EntityConfig } from "../../entity-rows/types";
+import { processEditorEntities } from "../process-editor-entities";
+
+const cardConfigStruct = struct({
+  type: "string",
+  title: "string?",
+  entity: "string?",
+  image: "string?",
+  camera_image: "string?",
+  camera_view: "string?",
+  state_iamge: "object?",
```

Comment:
The ui editor doesn't support `state_image`, so if the config contains `state_image` it should fallback to `yaml`.